_docs/concepts/policy-and-control/mixer-config.md

```diff
@@ -132,7 +132,7 @@ This configures an adapter that reports data to the Prometheus system. This adap
 require any custom parameters and so doesn't have a `params` stanza.
 
 Each adapter defines its own particular format of configuration data. The exhaustive set of
-adapters and their specific configuration formats can be found [here]({{home}}/docs/reference/config/mixer/adapters/index.html).
+adapters and their specific configuration formats can be found [here]({{home}}/docs/reference/config/mixer/adapters).
```

Comment:
here and everywhere else please add the missing trailing / to save 1 redirect (ideally that would be a lint actually)